qa-pipelines/tasks/run-test.sh

```diff
@@ -6,9 +6,6 @@ if [[ -z "${TEST_NAME:-}" ]] ; then
     exit 0
 fi
 
-# Set kube config from pool
-source "ci/qa-pipelines/tasks/lib/prepare-kubeconfig.sh"
```

Comment:
should this be sourced somewhere? Other than this removal, everything here looks good!

Reply:
That was unintentional; removed in faa6a9393783a7f0db1996402bfc29013ca26a39